resources/views/menu/admin_menu.blade.php

```diff
@@ -34,6 +41,36 @@
                 {
                     array_push($listItemClass,'active');
                 }
+
+                // Childrens Permission Checker
+                $self_prefix = str_replace('/', '\/', $options->user->prefix);
+                $slug = str_replace('/', '', preg_replace('/^\/'.$self_prefix.'/', '', $children->link()));
+                if ($slug != '') {
+                    // Get dataType using slug
+                    $dataType = $options->user->dataTypes->first(function ($value) use ($slug) {
+                        return $value->slug == $slug;
+                    });
+
+                    if ($dataType) {
+                        // Check if datatype permission exist
+                        $exist = $options->user->permissions->first(function ($value) use ($dataType) {
+                            return $value->key == 'browse_'.$dataType->name;
```

Comment:
This is checking the permissions directly instead of going through the auth system... Old way of doing it, needs updating.  I'll tack that onto my to-do list